devops/infrastructure/build-stage.yml

```diff
@@ -1,4 +1,4 @@
-#  Copyright � Microsoft Corporation
+#  Copyright � Microsoft Corporation
```

Comment:
What does this change mean?

Reply:
Nothing.  Bad symbol in the Readme sometimes get injected temporarily.